packages/cassowary/lib/cassowary.dart

```diff
@@ -3,6 +3,8 @@
 // found in the LICENSE file.
 
 /// An implementation of the Cassowary constraint solving algorithm in Dart.
+/// 
+/// To use, import `package:cassowary/cassowary.dart`.
```

Comment:
Sorry, I'm confused.

A) can a developer, today, import cassowary?
B) if so, what does that import line look like?